sdcm/cluster.py

```diff
@@ -3042,7 +3039,7 @@ def terminate_node(self, node):
         node.destroy()
 
     def get_db_auth(self):
-        if self.params.get('use_ldap_authorization') and self.ldap_configured:
+        if self.params.get('use_ldap_authorization') and self.params.get('are_ldap_users_on_scylla'):
             user = LDAP_USERS[0]
```

Comment:
Can we continue to using default 'cassandra' user even ldap is configured?
If SaslauthdAuthenticator is enabled, we still need to create 'cassandra' user in ldap server.

Reply:
@roydahan wants us to run the load as well with the LDAP configuration, but you should have no trouble, if the users are correctly created in in LDAP (and Scylla)